build-docs.sh

```diff
@@ -4,6 +4,8 @@ set -euo pipefail
 # render docs to docs/dist
 (cd docs && bash ./build-docs.sh)
 
+lerna ls > docs/dist/packages.txt
```

Comment:
Might not be sufficient to actually be able to implement our installation canaries. I would argue that we probably need the names of the modules for every package manager that we publish into. Otherwise we will need heuristics about converting the javascript names to other languages and that's not possible because there are special cases there.

Reply:
I think the .jsii files are a better artifact to publish to our docs website (or a .zip file with all the .jsii files). That will include all the needed information (and also all of our type system, which is useful for other purposes).